pkgs/applications/misc/lswt/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchFromSourcehut, nixos, wayland }:
+
+stdenv.mkDerivation rec {
+  pname = "lswt";
+  version = "1.0.4";
+
+  buildInputs = [ wayland ];
+
+  src = fetchFromSourcehut {
+    owner = "~leon_plickat";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-Orwa7sV56AeznEcq/Xj5qj4PALMxq0CI+ZnXuY4JYE0=";
+  };
+
+  makeFlags = [
+    "DESTDIR=${placeholder ''out''}"
+    "PREFIX="
+  ];
+
+  meta = with lib; {
+    description = "A command that lists Wayland toplevels";
+    homepage = "https://git.sr.ht/~leon_plickat/lswt";
+    license = licenses.gpl3;
+    maintainers = [ maintainers.edrex ];
```

Comment:
```suggestion
    maintainers = with maintainers; [ edrex ];
```

Reply:
> can we break out the maintainer change into a separate commit or pr?

i think i'll just wait for one of these PRs to land, and then amend the other so it doesn't have the maintainer-list.nix change, that's no problem. Auto conflict resolution may take care of it anyway since it's the same change in both branches.

Reply:
idk if the `with` is better. `with` introduces scope ambiguity which confuses code analysis tools, since it dumps the entire arg scope into the current scope, and AFAIK is considered somewhat harmful.

Reply:
this is just the default syntax for nixpkgs, since it reduces stutter for authors >2